### PR TITLE
Adding and removing members

### PR DESCRIPTION
**Pull request by mauzey1**

### CODE REVIEW

**Comment by mauzey1** [comment]
@downiec  I have added you to the team page of aims-site.  Waiting for merge approval.

**Comment by mauzey1** [comment]
@davis278 Here is a pull request for updating https://aims.llnl.gov/team.html.  Once I get your job title I will add another commit to add you to the page.

**Review by davis278 — APPROVED** [review]

**Review by davis278 — APPROVED** [review]

**Review by davis278 — APPROVED** [review]

**Review by sterlingbaldwin — APPROVED** [review]
Looks good to me.

**Pull request merged**
